### PR TITLE
Add python 3 support

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

**Comment by TomasTomecek** [comment]
PR made for @dhodovsk. (and to make @bkabrda happy too)

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - "2.7"
+  - "3.5"
```
Comment (bkabrda) [inline .travis.yml]: Let's also add 3.6!

Comment (TomasTomecek) [reply .travis.yml]: I omit 3.6 so we have faster CI runs.

Comment (TomasTomecek) [reply .travis.yml]: they run in parallel, nvm

**Comment by bkabrda** [comment]
LGTM now, merging. Thanks!

**Pull request merged**